example_analysis/run_analysis.sh

```diff
@@ -3,6 +3,10 @@
 # Set the path to the main Snakefile and the config file
 SNAKEFILE="../workflow/Snakefile"
 CONFIG_FILE="config/config.yml"
+RULEGRAPH_FILE="../images/brieflow_rulegraph.png"
+
+# Generate a rulegraph of the Snakefile
+# snakemake --snakefile "$SNAKEFILE" --configfile "$CONFIG_FILE" --rulegraph | dot -Gdpi=100 -Tpng -o "$RULEGRAPH_FILE"
```

Comment:
Not working? Why commented out?

Reply:
It works but takes a little bit of extra computation so I want to keep it commented out while we are developing Brieflow and are constantly running this script.

I added a note about this in https://github.com/cheeseman-lab/brieflow/pull/10/commits/958abc37b07cdcdc85d60e282f0ca6106201f43f